### ISSUE TITLE
error installing react-stockcharts v 0.2.3 from NPM 

### ISSUE THREAD

**Issue by raptoria** [rrag/react-stockcharts#18]
I'm trying to download react-stockcharts from behind a corporate firewall..(so this may be a proxy issue)

the problem occurs when npm attempts to download from a github account 

user@user-VirtualBox:~/react-flask$ npm install
      npm WARN package.json react-starter@0.0.0 No repository field.
      npm ERR! git clone --template=/home/user/.npm/_git-remotes/_templates --mirror git://github.com/exupero/saveSvgAsPng.git /home/user/.npm/_git-remotes/git-github-com-exupero-saveSvgAsPng-git-a9624716: Cloning into bare repository '/home/user/.npm/_git-remotes/git-github-com-exupero-saveSvgAsPng-git-a9624716'...
      npm ERR! git clone --template=/home/user/.npm/_git-remotes/_templates --mirror git://github.com/exupero/saveSvgAsPng.git /home/user/.npm/_git-remotes/git-github-com-exupero-saveSvgAsPng-git-a9624716: fatal: unable to connect to github.com:
      npm ERR! git clone --template=/home/user/.npm/_git-remotes/_templates --mirror git://github.com/exupero/saveSvgAsPng.git /home/user/.npm/_git-remotes/git-github-com-exupero-saveSvgAsPng-git-a9624716: github.com[0: 192.30.252.129]: errno=Connection timed out

Is there any workaround?

thanks!

**Comment by rrag** [comment]
Few questions

1. What is left after the failure? do you still have the react-stockcharts created under node_modules?

2. are you able to 
`npm install git://github.com/exupero/saveSvgAsPng.git#gh-pages`


I am not sure if this would work but worth a shot. since it appears the npm install from git seems to be blocked for you
do `npm install -S  save-svg-as-png` before `npm install -S react-stockcharts`

You will have some trouble with the save as png feature of `react-stockcharts`

**Comment by raptoria** [comment]
after the install, it created a folder react-stockcharts but none of it's dependencies were downloaded.

When I tried  npm install save-svg-as-png beforehand, it would still break on the same line, trying to access it from github. The only thing that worked was going into react-stockcharts, and changing the dependency to     "save-svg-as-png": "1.0.0" . I also tried "save-svg-as-png": https://github.com/exupero/saveSvgAsPng.git#gh-pages which is what a lot of other npm packages use when referencing github, but this failed. Is there a difference between using the NPM version and using the git version?
